articles/architecture-scenarios/application/mobile-api.md

```diff
@@ -9,30 +9,369 @@ beta: true
 
```

Comment:
Also add a table of contents dropdown with `toc: true` (we do this for long documents), see [here](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#front-matter)

It looks like that:

![image](https://user-images.githubusercontent.com/11715799/29118667-963f20ac-7d0b-11e7-8f10-7c20b62926b3.png)